widgetastic_manageiq/__init__.py

```diff
@@ -2859,6 +2859,7 @@ def select_by_visible_text(self, *items):
                 try:
                     # Added this as for some views(some tags pages) dropdown is separated from
                     # button and doesn't have exact id or name
+                    self.open()
```

Comment:
self.open is called before the loop is entered, do we need it called in both cases?

Reply:
I had to add it for something, need to check now

Reply:
looks like it works now without this additional open